lib/puppet/network/http.rb

```diff
@@ -5,6 +5,9 @@ module Puppet::Network::HTTP
   MASTER_URL_PREFIX = "/puppet"
   CA_URL_PREFIX = "/puppet-ca"
 
+  PUPPET_API_VERSIONS = "v3"
+  PUPPET_CA_API_VERSIONS = "v1"
+
```

Comment:
Would it make sense to name these two VERSIONS constants to match up with the corresponding `_URL_PREFIX` constants above? I.e. to make it more obvious that they go with those.

E.g.

```
MASTER_URL_PREFIX = "/puppet"
MASTER_URL_VERSIONS = "v3"

CA_URL_PREFIX = "/puppet-ca"
CA_URL_VERSIONS = "v1"
```

Naming is hard, so please feel free to ignore me :)